actors/cloudbroker/cloudbroker__computenode/methodclass/cloudbroker_computenode.py

```diff
@@ -145,16 +146,16 @@ def _get_stack_machines(self, stackId, fields=None):
         return machines
 
     @auth(['level2', 'level3'], True)
-    def maintenance(self, id, vmaction, **kwargs):
+    def maintenance(self, id, vmaction, force, **kwargs):
         """
         :param id: stack Id
         :param vmaction: what to do with vms stop or move
         :return: bool
         """
         title = ""
-        if vmaction not in ('move', 'stop'):
-            raise exceptions.BadRequest("VMAction should either be move or stop")
-
+        if vmaction not in ('migrate', 'stop'):
+            raise exceptions.BadRequest("VMAction should either be migrate, stop")
+        force = bool(int(force))
```

Comment:
you shouldnt have to do this atleast if you define force in the actor.spec properly